Chiazokam/Politico
# #163504412 create the get all parties endpoint

#### What does this PR do?
This PR tests and creates the get all parties Endpont

#### Description of Task to be completed?
- write tests for every test case
- create middleware to check if user is admin
- create the get all parties endpoint

#### How should this be manually tested?
- Clone the repo - `https://github.com/Chiazokam/Politico`
- Enter into the project directory - `cd Politico`
- switch to this branch - `checkout ft-database-admin-get-all-parties-163504412`
- Start the server `npm run start:dev`
- Open `Postman`
- On `postman` Send a `GET` request to `http://localhost:3000/api/v1/parties `
- You should receive a successful response

#### Any background context you want to provide?
Nil

#### What are the relevant pivotal tracker stories?
[#163504412](https://www.pivotaltracker.com/story/show/163504412)

### Images
![image](https://user-images.githubusercontent.com/26940294/51874465-55cfea80-2361-11e9-94e1-2dc22821ad3d.png)
